src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .map(pairs => pairs.split(':'))
+    .filter(pair => pair.length === 2)
+    .forEach(pair => {
```

Comment:
Краще використай метод reduce, щоб створити об'єкт і додати туди властивості

Reply:
Використай деструктуризацію
```suggestion
    .forEach(([key, value]) => {
```